emission/analysis/modelling/trip_model/greedy_similarity_binning.py

```diff
@@ -119,6 +119,11 @@ class label to apply:
         self.sim_thresh = config['similarity_threshold_meters']
         self.apply_cutoff = config['apply_cutoff']
         self.is_incremental = config['incremental_evaluation']
+        if config.get('clustering_way') is None:
```

Comment:
This is the case when other dependencies initiate `GreedySimilarityBinning`. We set the value to previous default ,i.e., 'origin-destination'.

Reply:
Note that it might be easier to add explanations as code comments, like we used to in the old days.